cve_bin_tool/checkers/__init__.py

```diff
@@ -96,6 +96,14 @@ def __new__(cls, name, bases, props):
         cls.VENDOR_PRODUCT = list(
             map(lambda vpkg: VendorProductPair(*vpkg), cls.VENDOR_PRODUCT)
         )
+        # Validate that vendor product pair is in lowercase
+        for items in cls.VENDOR_PRODUCT:
+            for vp in items:
+                if vp.islower() != True:
+                    raise AssertionError(
+                        "Checker %s has a VENDOR_PRODUCT string that is not lowercase"
```

Comment:
Please use fstrings here. (In case you missed it, here's where we ask for fstrings in the style guide: https://cve-bin-tool.readthedocs.io/en/latest/CONTRIBUTORS.html#style-guide-for-cve-bin-tool )